### PR TITLE
Update Workbox docs to 4.3.0

### PR DESCRIPTION
**Pull request by philipwalton**
**Target Live Date:** ASAP

- [x] This has been reviewed and approved by (@jeffposnick )
- [x] I have run `npm test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by philipwalton** [comment]
Ahh, thanks. Good catch!

File: src/content/en/tools/workbox/reference-docs/_redirects.yaml [hunk 1]
```diff
@@ -38,3 +38,6 @@ redirects:
 
 - from: /web/tools/workbox/reference-docs/v4.0.0/...
   to: /web/tools/workbox/reference-docs/v4.2.0/...
+
+- from: /web/tools/workbox/reference-docs/v4.2.0/...
```
Comment (jeffposnick) [inline src/content/en/tools/workbox/reference-docs/_redirects.yaml]: Now there's going to be a double-redirect for anyone who goes to `v4.0.0`, where it takes them to `v4.2.0`, and then where that takes them to `latest`.

What I've been doing with these redirects is not to use the `latest` alias at all, and just explicitly update any `vX.Y.Z` URLs to the explicit latest version in the `vX` major release.

Comment (philipwalton) [reply src/content/en/tools/workbox/reference-docs/_redirects.yaml]: The problem in this case is there's no `v4.3.0` directory.

**Comment by philipwalton** [comment]
Closing this in favor of #7526.

**Pull request closed without merging**